.github/workflows/changelog.yml

```diff
@@ -0,0 +1,15 @@
+name: Changelog checks
+
+on:
+    pull_request:
+        branches: [ main ]
+
+jobs:
+    build:
+        if: ${{ contains(github.event.pull_request.body, '[x] I''ve added new item into [Changelog](docs/Changelog.md) under [Unreleased]') }}
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Run changelog checker
+                uses: mwcodebase/versioning-checker@v2.2
```

Comment:
I know that it's a bit of an overkill in this case (and not even practiced in this repo, but I'm getting to it!) but you shouldn't use version tag when using github action https://julienrenaux.fr/2019/12/20/github-actions-security-risk/ so I would appreciate if you switched tag to a commit SHA
Can you point me to the source code of this action so that I can review how it works?

Reply:
Thank you for comment, I switched from tag to commit SHA and this is a action' [source code](https://github.com/electreos/versioning-checker).

btw, I'm quite new to github action, but perhaps I can try checking the changelog without using any 3rd party actions so if you have any suggestion, please let me know.

Reply:
Great! Thanks. If you want to implement that check by yourself then I would suggest reading up on [Github Actions documentation ](https://docs.github.com/en/actions) 
You can run shell commands like this https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun and that opens up almost everything that you would like to do
How about I'll merge this PR, and you will make a next one with a custom action implementation? Does that sounds good?

Reply:
that's sound good, thank you 👍